<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectKeyForm</class>
 <widget class="QWidget" name="SelectKeyForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>305</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>495</width>
    <height>305</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Выбор секретного ключа</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(&quot;Image/secret_success.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>137</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Путь к файлу секретного ключа:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:Tahoma;
font-style: normal;
font-size: 12pt;
font-weight: bold;

font-size:17px;
color:blue;</string>
   </property>
   <property name="text">
    <string>Выберите файл секретного ключа</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditKeyPath">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>265</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:center;
background-image: url(&quot;Image/ok.png&quot;);
background-repeat: no-repeat;
background-position: left 
</string>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>264</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:center;
background-image: url(&quot;Image/cancel.png&quot;);
background-repeat: no-repeat;
background-position: left 
</string>
   </property>
   <property name="text">
    <string>    Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectKeyPath">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:center;
background-image: url(&quot;Image/open-folder-yellow.png&quot;);
background-repeat: no-repeat;
background-position: left 
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
